MultiPage-Clone/index.html

```diff
@@ -4,6 +4,7 @@
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <link rel="stylesheet" href="css/main.css"/>
```

Comment:
I don't think this change belongs in this PR.  Did you commit this by mistake?

Reply:
Yes. That is by mistake this does not belong to wireframe.